src/ResourceManager/Sql/Commands.Sql/VulnerabilityAssessment/Cmdlet/VulnerabilityAssessmentSettings/GetAzureSqlDatabaseVulnerabilityAssessmentSettings.cs

```diff
@@ -0,0 +1,79 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Sql.Database.Model;
+using Microsoft.Azure.Commands.Sql.VulnerabilityAssessment.Cmdlet;
+using Microsoft.Azure.Commands.Sql.VulnerabilityAssessment.Model;
+using Microsoft.Azure.Commands.Sql.VulnerabilityAssessment.Services;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.VulnerabilityAssessmentSettings.Cmdlet
+{
+    /// <summary>
+    /// Returns the Vulnerability Assessment settings of a specific database.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "AzureRmSqlDatabaseVulnerabilityAssessmentSettings", SupportsShouldProcess = true),
```

Comment:
a ShouldProcess is not required on a Get... its required on any cmdlet that would change state though... Doers the get operation alter state?